src/gpu-compute/wavefront.cc

```diff
@@ -341,48 +340,6 @@ Wavefront::initRegState(HSAQueueEntry *task, int wgSizeInWorkItems)
                         wfSlotId, wfDynId, physSgprIdx,
                         task->privMemPerItem());
                 break;
-              case GridWorkgroupCountX:
```

Comment:
Are these removed in gfx90a?  Else why are you removing them?

Reply:
These might have been removed in GFX7 (?).  The bits have been repurposed it seems.  See bits 457:455 here: https://llvm.org/docs/AMDGPUUsage.html#code-object-v3-kernel-descriptor .

These must now always be zero, therefore the code will never be used.  I didn't want to leave the code there as it could cause the SGPR offsets to be incorrect